cli/src/list.rs

```diff
@@ -198,27 +253,40 @@ pub async fn list(current_crate: &Crate, is_verbose: bool) -> eyre::Result<()> {
         parser.visit_file(&syntax);
     }
 
+    if !is_verbose {
+        simple(&parser.functions, current_crate);
+        return Ok(());
+    }
+
```

Comment:
@quesabe let's rename to `project_url`. Also please get the root domain from `build_config().domain` (I added it recently).

Reply:
Done.